app/locales/es.js

```diff
@@ -156,13 +156,14 @@ export default {
   },
   resetPassword: {
     heading: 'Reset Password',
-    newPasswordFormDialog: 'Go ahead and enter a new password, then you can get back to doing some research.',
+    newPasswordFormDialog: 'Enter matching passwords that are at least 8 characters long so you can get back to doing some research.',
```

Comment:
I think the "matching passwords" is a bit weird. A confirmation field is quite normal, and I can't remember seeing it labeled like this. I'd be wondering "matching to what?".

Reply:
Perhaps "Enter the same password twice so you can get back to doing some research. Passwords need to be at least 8 characters long."?